doc/api/os.md

```diff
@@ -215,6 +215,14 @@ added: v2.3.0
 The `os.homedir()` method returns the home directory of the current user as a
 string.
 
+POSIX:
+Will use the `$HOME` environment variable if defined. Otherwise, it will use
+the effective user home directory.
+
+Windows:
+Will use the `USERPROFILE` environment variable if defined. Otherwise it
+will be the path to the root directory of the current user.
```

Comment:
Maybe s/root/home/ and "as returned by GetUserProfileDirectory()" at the end? The current phrasing isn't exactly wrong but I don't know if "root directory" is traditional Windows nomenclature.

Reply:
I do not know, libuv is calling `GetUserProfileDirectoryW()` method on windows, and looking at the [documentation](https://docs.microsoft.com/en-us/windows/desktop/api/userenv/nf-userenv-getuserprofiledirectoryw) "root directory" is mentioned as "notation", what do you think?

Reply:
MSDN calls it the "profile directory" which seems like a good name. Sounds reasonable?

Reply:
Sounds much much clear! I'll make those changes